models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md

```diff
@@ -19,37 +19,36 @@ This model is Mask-RCNN-based text detector with ResNet50 backbone and additiona
 
 | Metric                                        | Value     |
 |-----------------------------------------------|-----------|
-| Word spotting hmean ICDAR2015, w/o dictionary | 59.04%    |
+| Word spotting hmean ICDAR2015, without a dictionary | 59.04%    |
 | Detection hmean ICDAR2015                     | 87.09%    |
 | GFlops                                        | 185.169   |
 | MParams                                       | 26.497    |
 | Source framework                              | PyTorch\* |
 
-*Hmean Word spotting* is defined and measured according to
+*Hmean Word spotting* is defined and measured according to the
 [Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
 
 ## Performance
 
 ## Inputs
 
-1.	name: `im_data` , shape: [1x3x768x1280] - An input image in the format
-    [1xCxHxW]. The expected channel order is BGR.
-1.	name: `im_info`, shape: [1x3] - Image information: processed image height,
-    processed image width and processed image scale
+1.	Name: `im_data` , shape: [1x3x768x1280]. An input image in the [1xCxHxW] format. 
+    The expected channel order is BGR.
+2.	Name: `im_info`, shape: [1x3]. An input image with a processed image height,
```

Comment:
This isn't an image. It is _information_ about the input image, as the original description suggests.